app/src/main/java/com/example/androiddevchallenge/features/CountdownTimerScreen.kt

```diff
@@ -56,6 +56,7 @@ import androidx.lifecycle.viewmodel.compose.viewModel
 @Composable
 internal fun CountDownTimerScreen(countDownTimerViewModel: CountDownTimerViewModel = viewModel()) {
     val duration: String by countDownTimerViewModel.durationInMinutesAndSeconds.observeAsState("00:00")
```

Comment:
This could. be renamed to something like `formattedDuration` due to the introduction of a new duration variable,`remainingDuration`.